exec/java-exec/src/main/java/org/apache/calcite/jdbc/DynamicRootSchema.java

```diff
@@ -76,19 +72,19 @@ protected CalciteSchema getImplicitSubSchema(String schemaName,
    * @param schemaName the name of the schema
    * @param caseSensitive whether matching for the schema name is case sensitive
    */
-  public void loadSchemaFactory(String schemaName, boolean caseSensitive) {
+  private void loadSchemaFactory(String schemaName, boolean caseSensitive) {
     try {
       SchemaPlus schemaPlus = this.plus();
-      StoragePlugin plugin = getSchemaFactories().getPlugin(schemaName);
-      if (plugin != null && plugin.getConfig().isEnabled()) {
+      StoragePlugin plugin = storages.getPlugin(schemaName);
```

Comment:
What the difference in using `storages` instead of `getSchemaFactories`?

Reply:
`getSchemaFactories()` was an function to access `storages`. No need to use external assessor for the internal function, it is only adding not wanted overhead.